icssc/AntAlmanac
# Update required pnpm version

## Summary
- Update manifests to allow for the usage of `pnpm` 9.

## Test Plan
- Update `pnpm` to `9.x.x`. (Check with `pnpm -v`)
- Try to update dependencies with `pnpm install`
- Run app with `pnpm run start:aa`

## Issues

Closes #1007
